pyproject.toml

```diff
@@ -50,3 +55,16 @@ known-first-party = ["marray"]
 [tool.ruff.lint.flake8-tidy-imports]
 # Disallow all relative imports.
 ban-relative-imports = "all"
+
+[tool.coverage.run]
+source = ["marray"]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+exclude_lines = ["pragma: no cover", "if TYPE_CHECKING"]
+
+[tool.pytest.ini_options]
+filterwarnings = [
+    "error:::marray.*",
```

Comment:
Do you think it's safe to add:
`"ignore:invalid value encountered:RuntimeWarning"`
?
I don't think we'll get it because `marray` is doing something wrong, but it's easily generated in tests unless we're very careful about the arguments to functions (e.g. only positive values to square root, etc.)